docs/markdown/Using Pants/validating-dependencies.md

```diff
@@ -266,9 +266,10 @@ The previous example, using this alternative syntax for the selectors, would loo
 Glob syntax
 -----------
 
-The visibility rules are all about matching globs. There are two kinds of glob syntax: the simple glob syntax and the path glob syntax. Simple globs only support `*` to match anything and are otherwise case sensitive, and are used for all glob values except for the `path` part of a __target rule spec__. The path glob syntax is described further below.
+The visibility rules are all about matching globs. There are two wildcards, the `*` to match anything except `/` and the `**` to match anything including any `/`. (For paths that is non-recursive and recursive globbing respectively.)
+
+Globs are matched until the end of the value it is being applied to, so if there is no trailing wildcard (`*` or `**`) on the end of the path glob, it will match to the end of the value. An example where this is usefull is for matching on file names regardless of where in the project tree they are:
```

Comment:
```suggestion
A glob is matched until the end of the value it is being applied to, so if there is no trailing wildcard (`*` or `**`) on the end of the path glob, it will match to the end of the value. An example where this is useful is for matching on file names regardless of where in the project tree they are:
```